### PR TITLE
Mempool Goggles icon, tooltip, FAQ

### PR DESCRIPTION
**Pull request by mononaut**
Adds some polish to the mempool block filters added in #4500 by changing the icon, adding a "beta" tag and an info tooltip which links to a new FAQ entry.

I couldn't find a good "goggles" or "glasses" icon in the usual library, so this uses a custom SVG image.

See also #4510 and #4515.

Screenshot 2023-12-15 at 1 38 36 PM

Screenshot 2023-12-15 at 1 27 32 PM

### CODE REVIEW

File: frontend/src/app/shared/components/svg-icons/goggles.component.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
```
Comment (softsimon) [inline frontend/src/app/shared/components/svg-icons/goggles.component.ts:1]: We can't use the current `svg-images-component.html` ?

Comment (Jean68172) [reply frontend/src/app/shared/components/svg-icons/goggles.component.ts:1]: Yes

Comment (mononaut) [reply frontend/src/app/shared/components/svg-icons/goggles.component.ts:1]: yes absolutely. I didn't realize what that component did when I checked it earlier - I thought it was just for the bitcoin/liquid network logos.

**Comment by mononaut** [comment]
Rebased on current master and changed the svg icon to use the existing svg-images component

**Review by softsimon — APPROVED** [review]
tested ACK @ [78857102f]

**Pull request merged**